library/ssl_tls13_client.c

```diff
@@ -422,9 +535,47 @@ int mbedtls_ssl_tls13_handshake_client_step( mbedtls_ssl_context *ssl )
             break;
 
         case MBEDTLS_SSL_SERVER_HELLO:
-            // Stop here : we haven't finished whole flow
-            ret = MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE;
-            mbedtls_ssl_handshake_set_state( ssl, MBEDTLS_SSL_ENCRYPTED_EXTENSIONS );
+            ret = ssl_tls1_3_read_server_hello( ssl );
+            break;
+
+        case MBEDTLS_SSL_ENCRYPTED_EXTENSIONS:
+            ret = ssl_tls1_3_read_encrypted_extensions( ssl );
+            break;
+
+        case MBEDTLS_SSL_CERTIFICATE_REQUEST:
+            ret = ssl_tls1_3_read_certificate_request( ssl );
+            break;
+
+        case MBEDTLS_SSL_SERVER_CERTIFICATE:
+            ret = ssl_tls1_3_read_server_certificate( ssl );
+            break;
+
+        case MBEDTLS_SSL_CERTIFICATE_VERIFY:
+            ret = ssl_tls1_3_read_certificate_verify( ssl );
+            break;
+
+        case MBEDTLS_SSL_SERVER_FINISHED:
+            ret = ssl_tls1_3_read_server_finished( ssl );
```

Comment:
I just check this function in tls1.3 prototype, there it's named mbedtls_ssl_finished_in_process(), and is defined in ssl_tls13_generic.c.   So, do we need to add one wrap function ssl_tls1_3_read_server_finished() in this file?

Reply:
This PR is for reduce possible conflicts in `MVP` .  Most of our PRs depends on others, and the PRs are parallel work. If merged one , all of others must re-work and resolve conflict. 
Base on my experience , if all guys got a fixed place to put codes , there will be no conflict .  

>>So, do we need to add one wrap function ssl_tls1_3_read_server_finished() in this file?
That can be done at final of `MVP` . I think so . In my view , most of code will be overwrite when MVP done. 
But in MVP stage, I do not think change the line is good idea.

Reply:
Can this topic be closed?